src/setupTests.js

```diff
@@ -0,0 +1,3 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+configure({ adapter: new Adapter() });
```

Comment:
@BrianSerem please add an end of file to this file

Reply:
I believe there is an EOL, files without have little red pointer arrows at the bottom.

Reply:
Thanks for the clarification @BrianSerem